pos_financial_risk/static/src/js/ProductScreen.js

```diff
@@ -0,0 +1,31 @@
+odoo.define("pos_financial_risk.ProductScreen", function (require) {
```

Comment:
please use the new method to define the js files, this will be more flexible for porting to a new version of odoo in the future

```js
/** @odoo-module **/


import ProductScreen from "point_of_sale.ProductScreen";
import Registries from "point_of_sale.Registries";
import { _t } from "web.core";

export const ProductScreenRisk = (ProductScreen) =>
```
and etc